src/main/java/org/gitlab4j/api/ApplicationSettingsApi.java

```diff
@@ -1,23 +1,20 @@
 package org.gitlab4j.api;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import jakarta.ws.rs.core.Response;
 import java.text.ParseException;
 import java.util.Iterator;
-
-import javax.ws.rs.core.Response;
-
-import org.gitlab4j.api.models.Setting;
 import org.gitlab4j.api.models.ApplicationSettings;
+import org.gitlab4j.api.models.Setting;
 import org.gitlab4j.api.utils.ISO8601;
 
-import com.fasterxml.jackson.databind.JsonNode;
-
 /**
  * This class implements the client side API for the GitLab Application Settings API.
  * See <a href="https://docs.gitlab.com/ee/api/settings.html">Application Settings API at GitLab</a> for more information.
  */
 public class ApplicationSettingsApi extends AbstractApi {
 
-    public ApplicationSettingsApi(GitLabApi gitLabApi) {
+    public ApplicationSettingsApi(final GitLabApi gitLabApi) {
```

Comment:
Why did you make all the method argument explicitly final?

I think the compiler was already treating those as implicitly final